### ISSUE TITLE
[Design] Extraneous bottom-border on title of open accordions

### ISSUE THREAD

**Issue by tomstringer** [unimelb/design-system-issues#7]
Accordions display a bottom-border under the title element. When open, this should not be displayed.

In Gen1, the `bottom-border` is attached to the parent `li` element only. In the open state, it also inherits a new colour.

![Screen Shot 2019-10-31 at 12 01 10 pm](https://user-images.githubusercontent.com/924501/67910520-23e10280-fbd7-11e9-9071-dadc83d5488c.png)

```
.uomcontent ul.accordion>li.accordion__visible {
  margin-top: -1px;
  border-top: 1px solid;
  border-color: #006fdc;
},
.uomcontent ul.accordion>li.accordion__visible .accordion__title {
  color: #006fdc;
}
```
In Gen3, both the parent `li` element and the `accordion__title` span have a `border-bottom` style.

![Screen Shot 2019-10-31 at 12 08 59 pm](https://user-images.githubusercontent.com/924501/67910550-3eb37700-fbd7-11e9-9835-644ade30d791.png)

```
uom.css:1:194146.uomcontent ul.accordion>li {
  border-bottom: 1px solid #cde6ff;
  list-style-type: none;
  margin: 0;
  padding-bottom: 0;
  max-width: 2000px;
  width: 100%;
},
.uomcontent ul.accordion>li.accordion__visible .accordion__title {
  border-bottom: 1px solid #cde6ff;
}
```

- [ ] Remove border-bottom from `accordion__visible.accordion__title` element
- [ ] Provide a differentiated colour for `accordion__visible` borders

**Comment by jaredh-um** [comment]
@tomstringer have you got a URL for this one? The documentation site doesn't have a similar example

**Comment by tomstringer** [comment]
Oh sorry @jaredh-um 

https://web.unimelb.edu.au/components/accordion/ for DDS

You can use https://cms.unimelb.edu.au/controlled-environment/erb-workstream1/services/arts/arts-research-office/_nocache with the Gen3 design applied for an example.

**Comment by jaredh-um** [comment]
Hi @makenosound can you please make the adjustments as per Tom's notes above - colour for point 2 is Navy Main #094183 which will provide sufficient contrast for this element.
